src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -34,6 +35,7 @@ def __init__(self, app, api, config, mount):
         self.reqmgr_db_service = RequestDBWriter(self.reqmgr_db, couchapp = "ReqMgr")
         # this need for the post validtiaon 
         self.reqmgr_aux_db = api.db_handler.get_db(config.couch_reqmgr_aux_db)
+        self.gq_service = WorkQueue("%s/%s" % (config.couch_host, config.couch_workqueue_db))
```

Comment:
You are right, I was think we will all use url format with db name attached, but it looks silly. I will change that.